### ISSUE TITLE
MobileBERT finetuning hyperparameters

### ISSUE THREAD

**Issue by asharma20** [google-research/google-research#287]
In the MobileBERT paper, it's mentioned that the optimization hyperparameters (batch sizes, learning rates, number of epochs) are searched to choose the model. What are the optimal hyperparameters for the available tasks?

**Comment by saberkun** [comment]
Usually the good parameters for squad v1.1 is around 5 epochs, learning rate 4e-5, batch size 32.
The search range for squad is small. +-2 for epochs and LR.
But for GLUE, some tasks need to run longer.

**Comment by asharma20** [comment]
Do you have some more specifics for the hyperparameters for the GLUE tasks? Also it seems like the results for CoLA, MRPC, and RTE vary quite a bit (~3%) between runs. Is there a reason for that and is there a way to get more stable results?

**Comment by saberkun** [comment]
We found the pooler head is harmful to GLUE tasks sometime. You may try to do not initialize from the pooler dense or just do not have that inside the finetuning model.

**Comment by changquanyou** [comment]
Finetuneing Sst-2 Task on moblieBert, The dev acc is **0.8027523**, It's a huge difference 
hyperparameters: 
batch_size=64,
length=256, 
epoch=5,
learning rate=5e-5
